docker_services_cli/docker-services.yml

```diff
@@ -47,6 +47,25 @@ services:
     ports:
       - "9200:9200"
       - "9300:9300"
+  opensearch:
+    image: opensearchproject/opensearch:${OPENSEARCH_VERSION}
+    restart: "unless-stopped"
+    environment:
+      - bootstrap.memory_lock=true
+      - "OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m"
+      - discovery.type=single-node
+      - plugins.security.disabled=true
```

Comment:
OS 1.x enables SSL and auth (`admin:admin`) per default... which i think is also the case with ES 8.x